Frontend/src/index.html

```diff
@@ -3,9 +3,9 @@
 <head>
   <meta charset="utf-8">
   <title>TimeTracker</title>
+  <link rel="shortcut icon" href="/assets/site-icon.png" type="image/x-icon" />
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
```

Comment:
Both lines `rel='icon'` and `rel='shortcut icon'` should be used. Most mobile browsers will actually ignore the `shortcut icon`. Probably just convert the png to an ico with some online tool and name it `favicon.ico` this makes browsers happy.